### ISSUE TITLE
Allow Accept header to be customized

### ISSUE THREAD

**Issue by thegranddesign** [fenichelar/ember-simple-auth-token#159]
The configuration has a `headers` option, which is great, but unfortunately the [makeRequest method overrides the Accept header](https://github.com/jpadilla/ember-simple-auth-token/blob/master/addon/authenticators/jwt.js#L302) so a custom Accept header is not possible.

I'd be willing to PR this.

**Comment by thegranddesign** [comment]
I see what the problem is.  The references to `headers` [here](https://github.com/jpadilla/ember-simple-auth-token/blob/master/addon/authenticators/jwt.js#L304-L306) needs to be `this.headers`.

**Comment by thegranddesign** [comment]
What about something like this?

``` js
if(this.headers['Accept'] === null || this.headers['Accept'] === undefined) {
  xhr.setRequestHeader('Accept', settings.accepts.json);
}
```

**Comment by lllama** [comment]
:+1: At the moment I have an API that talks JSON-API except for authentication due to this issue.

**Comment by jpadilla** [comment]
@thegranddesign looks good, a PR would definitely be welcome.

**Comment by joshsmith** [comment]
@thegranddesign do you still plan to PR this?

**Comment by mmwebster** [comment]
Just ran into the same issue. @jpadilla I'll start working on a PR if it's still welcome.

**Comment by joshsmith** [comment]
Is this closed now by #178?

**Comment by jpadilla** [comment]
@JoshSmith yes!
